### PR TITLE
Add isort to pre-commit hooks.

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR adds the isort (and related seed-isort-config) hooks to the pre-commit configuration. It also reformats the existing code based on those hooks. Lastly it pins coverage at < 5.0 due to current incompatibility in the coveralls package. Please see the issue https://github.com/coveralls-clients/coveralls-python/issues/203 for the status of support.

## 💭 Motivation and Context
Since we already use black to standardize code format it makes sense to automatically format imports in our Python project in a similar way. This is done using the [isort tool](https://github.com/timothycrosley/isort) and enhanced with the [seed-isort-config](https://github.com/asottile/seed-isort-config) tool to statically populate third party packages in the isort configuration file `.isort.cfg`. The caveat is that first party packages must be explicitly defined in the configuration file to correctly be separated.

Lastly when doing a fresh `pip install -r requirement-test.txt` the coverage version installed is 5.0 which is not currently supported by coveralls. This is pinned in `setup.py` until support is added in coveralls.

## 🧪 Testing
I tested it on this project and did some tests using the `cyhy-feeds` project to see how it would work on a project with more code involved.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[settings]
+combine_star=true
+force_sort_within_sections=true
+
+import_heading_stdlib=standard python libraries
```
Comment (jsf9k) [inline .isort.cfg:5]: I'd like to see the first word of these capitalized (e.g. Standard python libraries), but I don't think it's critical.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I think it makes sense to add this to the [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) instead.  I'd like this import sorting sauce to trickle down to the python test code in [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role), for example.

**Review by dav3r — APPROVED** [review]
LGTM - excellent idea @mcdonnnj 👍

**Comment by mcdonnnj** [comment]
Per feedback from @jsf9k I added the `seed-isort-config` and `isort` hooks upstream in https://github.com/cisagov/skeleton-generic/pull/28 and so the changes in this PR are now represented in https://github.com/cisagov/skeleton-python-library/pull/30

**Pull request closed without merging**